mopidy/config/types.py

```diff
@@ -279,7 +279,7 @@ class Port(Integer):
 
     def __init__(self, choices=None, optional=False):
         super().__init__(
-            minimum=0, maximum=2 ** 16 - 1, choices=choices, optional=optional
+            minimum=0, maximum=2**16 - 1, choices=choices, optional=optional
```

Comment:
This is very unrelated and should be another PR.

Reply:
What about a pr that removes flake8 and black running on the master branch Ci? I'm getting tired of these.

Reply:
> What about a pr that removes flake8 and black running on the master branch Ci?

It would be better to pin the versions we're using and set up Dependabot or Renovatebot to keep them up to date for us. IMO using poetry would make this easier.

Reply:
Or that. To be clearer, I am only talking about doing this on master branch.